.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["amex"]
+  "presets": ["amex"],
+  "plugins": [
+    [
+      "@babel/transform-runtime"
```

Comment:
Why did you decide to add this?

Reply:
When using our DX Integration Sandboxes, NextJS does not transpile dependencies like we do. Therefore we must compile down. This will be replaced with a Rollup based config.